client/my-sites/exporter/index.jsx

```diff
@@ -36,7 +35,6 @@ function mapStateToProps( state ) {
 		downloadURL: state.siteSettings.exporter.downloadURL,
 		didComplete: getExportingState( state, siteId ) === States.COMPLETE,
 		didFail: getExportingState( state, siteId ) === States.FAILED,
-		showGuidedTransferOptions: config.isEnabled( 'manage/export/guided-transfer' ),
```

Comment:
I think we should keep the config stuff here and not move it into the component. Maybe what we need to do here is change the name from `showGuidedTransferOptions` to `isCalypsoGuidedTransferEnabled` or something that indicates it will use the wp-admin flow instead of the calypso flow.